test/over_react_test/jacket_test.dart

```diff
@@ -27,24 +27,22 @@ main() {
   group('mount: renders the given instance', () {
     group('attached to the document', () {
       group('and unmounts after the test is done', () {
-        TestJacket jacket;
+        late TestJacket jacket;
 
         setUp(() {
-          expect(document.body.children, isEmpty);
+          expect(document.body!.children, isEmpty);
         });
 
         tearDown(() {
-          expect(document.body.children, isEmpty);
+          expect(document.body!.children, isEmpty);
           expect(jacket.isMounted, isFalse);
-
-          jacket = null;
```

Comment:
Without reassigning jacket to null, there's a risk that if jacket doesn't get reassigned in a test, it could use stale values from previous tests. Perhaps we should just make jacket local variables in the tests, and move the `expect(jacket.isMounted, isFalse);` to within each test?